### DISCUSSION TITLE
Upgrade from v3.4 to v3.5

### DISCUSSION THREAD

**Discussion by marekalgoud** [alpinejs/alpine#2383] [1. Help] [answered]
Hello,

I'm trying to update my code to the last version of apline as I would like to use the $data feature, but  when I update the package version, I'm facing this issue evrywhere I use alpine : 

```Alpine Expression Error: func(...).catch is not a function```

Am I the only one who as this problem ?

Thanks for your help and your amazing work !

Marek

**Comment by SimoTod** [comment, marked as answer]
https://github.com/alpinejs/alpine/discussions?discussions_q=catch+is+not+a+function

**Reply by SimoTod** [reply]
just pick one, they mostly have the same answer, your build is targetting old browsers

**Reply by marekalgoud** [reply]
Thanks for your answer, I guess I have to improve my search skill :)

**Reply by SimoTod** [reply]
😂  no worries
